### ISSUE TITLE
[npm 3] Cannot find module 'app/node_modules/rnpm/node_modules/rnpm-plugin-link'

### ISSUE THREAD

**Issue by xxsnakerxx** [rnpm/rnpm#52]
rnpm installed as local dependency, launched via `npm run rn-link` (__node_modules/.bin/rnpm link__ in pjson script) and got

Error: Cannot find module 'app/node_modules/rnpm/node_modules/rnpm-plugin-link'
    at Function.Module._resolveFilename (module.js:327:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:355:17)
    at require (internal/module.js:13:17)
    at app/node_modules/rnpm/src/getCommands.js:12:3
    at Array.map (native)
    at getCommands (app/node_modules/rnpm/src/getCommands.js:23:27)
    at Object.<anonymous> (app/node_modules/rnpm/bin/cli:8:47)
    at Module._compile (module.js:399:26)
    at Object.Module._extensions..js (module.js:406:10)

__rnpm -v__ `1.3.4`
__npm -v__ `3.5.2`

I temporarily fixed this by changing `node_modules` to `..` [HERE](https://github.com/rnpm/rnpm/blob/master/src/getCommands.js#L12)

**Comment by Kureev** [comment]
Well, we haven't expected somebody will use rnpm as a local dependency, but I can easily add an additional path (as you mentioned above) to `getCommands` :wink:

**Comment by Kureev** [comment]
Should be fixed in the master branch since #54 has been merged. We'll bump a npm version soon.

**Comment by andycaramba** [comment]
The same issue with npm 3.5.3 and node_modules in $HOME
```javascript
> $ rnpm -v                                                                                                                                                         [±master ●●●]
module.js:340
    throw err;
    ^

Error: Cannot find module '/Users/andrey/node_modules/rnpm/node_modules/rnpm-plugin-link'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at /Users/andrey/node_modules/rnpm/src/getCommands.js:12:3
    at Array.map (native)
    at getCommands (/Users/andrey/node_modules/rnpm/src/getCommands.js:22:27)
    at Object.<anonymous> (/Users/andrey/node_modules/rnpm/bin/cli:8:47)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
```

```
> $ ls -la $HOME/node_modules/ | grep rnpm
drwxr-xr-x   15 andrey staff  510 янв  5 22:22 rnpm/
drwxr-xr-x   12 andrey staff  408 янв  5 22:22 rnpm-plugin-link/
```

```
> $ ls -la $HOME/node_modules/rnpm/node_modules/
total 0
drwxr-xr-x 13 andrey staff 442 янв  5 22:22 ./
drwxr-xr-x 15 andrey staff 510 янв  5 22:22 ../
drwxr-xr-x  4 andrey staff 136 янв  5 22:22 .bin/
drwxr-xr-x  9 andrey staff 306 янв  5 22:22 deep-extend/
drwxr-xr-x  8 andrey staff 272 янв  5 22:22 fs-extra/
drwxr-xr-x  6 andrey staff 204 янв  5 22:22 got/
drwxr-xr-x  9 andrey staff 306 янв  5 22:22 graceful-fs/
drwxr-xr-x  6 andrey staff 204 янв  5 22:22 latest-version/
drwxr-xr-x  6 andrey staff 204 янв  5 22:22 package-json/
drwxr-xr-x 12 andrey staff 408 янв  5 22:22 rc/
drwxr-xr-x  6 andrey staff 204 янв  5 22:22 repeating/
drwxr-xr-x  7 andrey staff 238 янв  5 22:22 strip-json-comments/
drwxr-xr-x  6 andrey staff 204 янв  5 22:22 update-notifier/
```

**Comment by grabbou** [comment]
Released under 1.3.5. Please see if it fixes your issue.

**Comment by andycaramba** [comment]
The issue is gone. Thanks.
